src/components/shared/IdleSearchField.css

```diff
@@ -29,17 +29,18 @@
 }
 
 .idleSearchFieldButton {
+  position: absolute;
+  right: 16px;
   width: 11px;
   height: 11px;
+
   overflow: hidden;
   background: url(../../../res/searchfield-cancel.svg) top left no-repeat;
   background-size: contain;
   border: 0;
   padding: 0;
   vertical-align: middle;
   color: transparent;
-  position: relative;
-  margin-left: -16px;
```

Comment:
Because of this margin-left, the box for the input was not properly positioned (despite being displayed correctly) and as a result the positioning of the string search stack was not correct.

So I used absolute positioning instead which looks as good and doesn't have the same issue.